repository/migrations/postgres/2021-09-17T10:00_create_requisition_table/up.sql

```diff
@@ -0,0 +1,23 @@
+-- Create requisition table.
+
+CREATE TYPE requisition_type AS ENUM ('REQUEST', 'RESPONSE');
+CREATE TYPE requisition_status AS ENUM ('DRAFT', 'NEW', 'SENT', 'FINALISED');
+
+CREATE TABLE requisition (
+    id TEXT NOT NULL PRIMARY KEY,
+    requisition_number BIGINT NOT NULL,
+    store_id TEXT NOT NULL REFERENCES name(id),
```

Comment:
Looks like name and store might be the wrong way around in this sql?

```suggestion
    store_id TEXT NOT NULL REFERENCES store(id),
    name_id TEXT NOT NULL REFERENCES name(id),
```

Reply:
Thanks James ! Nicely spotted